src/main/scala/beam/utils/csv/conversion/Xml2CsvFileConverter.scala

```diff
@@ -0,0 +1,19 @@
+package beam.utils.csv.conversion
+
+import java.io.File
+
+trait Xml2CsvFileConverter {
+  protected def fields: Seq[String]
+  protected lazy val header: Iterator[String] = Iterator(fields.mkString(FieldSeparator), LineSeparator)
```

Comment:
Why don't leave it as `def header`? Then you don't need to do the trick with `lazy val` because of init order